controllers/jobs.js

```diff
@@ -0,0 +1,25 @@
+const Job = require("../models/Job");
+const {StatusCodes} = require("http-status-codes");
+const {BadRequestError, NotFoundError} = require("../errors");
+
+const getAllJobs = async (req, res) => res.send("get all jobs");
+
+const getJob = async (req, res) => res.send("get job");
+
+const createJob = async (req, res) => {
+  req.body.createdBy = req.user.userId;
+  const job = await Job.create(req.body);
+  res.status(StatusCodes.CREATED).json({job});
+};
+
+const updateJob = async (req, res) => res.send("update job");
+
+const deleteJob = async (req, res) => res.send("delete job");
+
```

Comment:
Async is sued when you need to use await , while it works if you put async keyword in function but  
it is not that useful to put 'async' keyword if you do not use 'await' or need to return promise object. In this case,  you're not required.

Reply:
Thank you for noticing and pointing that out. The `06-jobs-api` segment of the course has two parts spanning week 9 and week 10. The `getAllJobs`, `getJob`, `updateJob` and `deleteJob` controller functionalities had not been implemented in week 9, the progress of which is what you have seen here. The code as you see here is done per the video while still in week 9 and would be fully implemented asynchronous functions in week 10, as could be seen in the next PR.